lib/content-services/src/lib/permission-manager/components/mat-chip.directive.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* tslint:disable:no-input-rename  */
+import { Directive, ElementRef, Renderer2 } from '@angular/core';
+
+@Directive({
+    selector: '[adf-list-a11y]'
+})
+export class MatChipDirective {
+    constructor(private el: ElementRef, private renderer: Renderer2) {}
+
+    ngAfterViewInit() {
+        setTimeout(() => {
+            this.renderer.removeAttribute(this.el.nativeElement, 'role');
```

Comment:
what would be the difference with what we do [here](https://github.com/Alfresco/alfresco-ng2-components/pull/5416/files#diff-a19aedfa9dcd553f1e38dedc522a1f02R29-R31)

Reply:
If we end up making Chris' change global, then we would not need this specifically. We would just need to add the attribute he uses for selector on this and it should work